$OpenBSD: patch-configure_ac,v 1.1.1.1 2015/10/30 17:18:53 landry Exp $

Check for POSIX_TERMIOS and define HAVE_TERMIOS_H for sprompt.cpp
to not echo passwords.

--- configure.ac.orig	Fri Aug 14 22:28:11 2015
+++ configure.ac	Thu Sep 10 22:33:32 2015
@@ -26,6 +26,12 @@ AC_GNU_SOURCE
 AC_HEADER_SYS_WAIT
 AC_FUNC_MMAP
 
+AC_SYS_POSIX_TERMIOS
+if test "$ac_cv_sys_posix_termios" = "yes"
+then
+  AC_DEFINE([HAVE_TERMIOS_H], [1], [termios.h])
+fi
+
 dnl Find C compiler
 dnl AC_PROG_CC_C99
 
